src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -284,6 +290,8 @@
    ["-F" "--facts FACTS" "Path to a directory containing sample JSON facts (files must end with .json)"]
    ["-C" "--catalogs CATALOGS" "Path to a directory containing sample JSON catalogs (files must end with .json)"]
    ["-R" "--reports REPORTS" "Path to a directory containing sample JSON reports (files must end with .json)"]
+   ["-A" "--archive ARCHIVE" "Path to a PuppetDB export tarball. Incompatible with -C, -F, or -R, or -D"]
+   ["-D" "--default" "Use the sample data in acceptance/benchmark. Incompatible with -C, -F, -R, or -A"]
```

Comment:
it will, the data just doesn't exist yet.

I held off for now because the data for reports will change when the metrics and logs patches land, and we already have some sample data from that work that will be perfect.